SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditor.h

```diff
@@ -17,7 +17,14 @@ class SPATIALGDKEDITOR_API FSpatialGDKEditor
 	{
 	}
 
-	bool GenerateSchema(bool bFullScan);
+	enum SchemaGenerationMethod
+	{
+		InMemoryAsset,
+		FullAssetScan,
```

Comment:
I think we should replace the current Full Scan with the new cook and generate, keeping it called "Full Scan". This issue is mainly hitting non-technical users so I think adding an extra option will be more confusing.